app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Consider using parentheses for importing multiple classes/modules to improve readability, as suggested in the checklist.